src/modules/transaction/components/txnDataTables/SdaExchangeTxnDataTable.vue

```diff
@@ -0,0 +1,311 @@
+          "
+          headerStyle="width:110px;text-transform:uppercase"
+        >
+          <template #body="{ data }">
+            <span class="text-xs">{{ convertLocalTime(data.timestamp) }}</span>
+          </template>
+        </Column>
+        <Column
+          field="typeName"
+          header="Type"
+          headerStyle="width:110px;text-transform:uppercase"
+          v-if="wideScreen"
+        >
+          <template #body="{ data }">
+            <span class="text-xs">{{ data.typeName }}</span>
+          </template>
+        </Column>
+        <Column
+          field="block"
+          header="Block"
+          v-if="
+            selectedGroupType === transactionGroupType.CONFIRMED && wideScreen
+          "
+          headerStyle="width:110px;text-transform:uppercase"
+        >
+          <template #body="{ data }">
+            <router-link
+              :to="{ name: 'ViewBlock', params: { blockHeight: data.block } }"
+              class="text-blue-600 hover:text-blue-primary hover:underline text-xs"
+              >{{ data.block }}</router-link
+            >
+          </template>
+        </Column>
+        <Column
+          header="TX FEE"
+          v-if="
+            selectedGroupType === transactionGroupType.CONFIRMED && wideScreen
+          "
+          headerStyle="width:110px;text-transform:uppercase"
+        >
+          <template #body="{ data }">
+            <div class="text-xs">
+              {{ data.fee }}
+              <b v-if="data.fee == 0 || data.fee > 0">{{ nativeTokenName }}</b>
+            </div>
+          </template>
+        </Column>
+        <Column 
+          header="SDA(Get/Give)" headerStyle="width:150px" 
+          v-if="wideScreen">
+        <template #body="{ data }">
+          <div v-if="checkOtherSDA(data.sdaExchange)">
+            <span
+              v-for="(sdaName, index) in displaySDAs(data.sdaExchange)"
+              :key="index"
+            >
+              <router-link
+                :to="{ name: 'ViewAsset', params: { id: sdaName.name } }"
+                class="text-blue-600 hover:text-blue-primary flex hover:underline"
+                >{{ sdaName.name }}</router-link
+              >
+              {{ sdaName.amount }}
+            </span>
+          </div>
+        </template>
+      </Column>
+        <Column
+        header="Duration"
+        headerStyle="width:40px;text-transform:uppercase"
+        v-if="wideScreen"
+      >
+        <template #body="{ data }">
+          <span
+            class="text-xs font-bold"
+            v-tooltip.left="
+              { value: `<tiptext>Approximately ${durationTime(getDuration(data.sdaExchange))} Day${(durationTime(getDuration(data.sdaExchange)) > 1 ? 's' : '')}</tiptext>` , escape: true }
+            "
+            >{{
+              getDuration(data.sdaExchange)
+                ? getDuration(data.sdaExchange) + " Block" + (getDuration(data.sdaExchange) > 1 ? "s" : "")
```

Comment:
usage of getDuration is wrong, it suppose to be have own duration for every offer